src/External-contrib/NGFC/NIST.py

```diff
@@ -0,0 +1,206 @@
+##############################################################################
```

Comment:
At the very least, you should be able to import the NIST module from core and only overload the things you changed, rather than writing a new module in full.

Reply:
My change in this file is to the calculation of enthalpy. I found that when using the framework's NIST module the solutions to Gibbs reactors are not correct. I believe this is because the equation is calculating the enthalpy change from the reference state, but the gibbs minimization needs the actual value. This might be something we want to correct in core, but I am unsure how my modifications will affect other flowsheets. What do you think?

The two changes I made are the removal of H and the removal of the tr's. 
This is the equation from core.
```
h = (cobj.cp_mol_ig_comp_coeff_A*(t-tr) +
    (cobj.cp_mol_ig_comp_coeff_B/2)*(t**2-tr**2) +
    (cobj.cp_mol_ig_comp_coeff_C/3)*(t**3-tr**3) +
    (cobj.cp_mol_ig_comp_coeff_D/4)*(t**4-tr**4) -
     cobj.cp_mol_ig_comp_coeff_E*(1/t-1/tr) +
     cobj.cp_mol_ig_comp_coeff_F -
     cobj.cp_mol_ig_comp_coeff_H)
```
And this is my modification.
```
h = (cobj.cp_mol_ig_comp_coeff_A*(t) +
    (cobj.cp_mol_ig_comp_coeff_B/2)*(t**2) +
    (cobj.cp_mol_ig_comp_coeff_C/3)*(t**3) +
    (cobj.cp_mol_ig_comp_coeff_D/4)*(t**4) -
     cobj.cp_mol_ig_comp_coeff_E*(1/t) +
     cobj.cp_mol_ig_comp_coeff_F)
```

Reply:
This issue was fixed in a previous PR - you should be able to use the base NIST library now. There is logic built in that excludes the `-H` term unless you explicitly say that heat of formation should not be included.